Dockerfile

```diff
@@ -97,9 +97,10 @@ RUN /download-frozen-image-v2.sh /build \
         busybox:latest@sha256:95cf004f559831017cdf4628aaf1bb30133677be8702a8c5f2994629f637a209 \
         busybox:glibc@sha256:1f81263701cddf6402afe9f33fca0266d9fff379e59b1748f33d3072da71ee85 \
         debian:buster@sha256:46d659005ca1151087efa997f1039ae45a7bf7a2cbbe2d17d3dcbda632a3ee9a \
+        debian:bullseye@sha256:7190e972ab16aefea4d758ebe42a293f4e5c5be63595f4d03a5b9bf6839a4344 \
```

Comment:
Wondering if we should replace the `:buster` variant (and switch the `buildpack-deps` image accordingly as well)

@tianon wdyt?

Reply:
Depends on how we use this image -- if we install packages, Bullseye is still a pretty heavily moving target (whereas Buster is "stable"), so using Bullseye will be more likely to introduce unrelated failures.

Reply:
Looks like we don't typically invoke `apt-get`, so it'd probably be fine to update to `bullseye` for both, since if the image upstream built successfully there's a high chance it'll work for what's used here (which will be pinned by digest anyhow).

Reply:
I added a commit to replace use of `debian:buster` with `debian:bullseye`.